aicsimageio/writers/ome_tiff_writer.py

```diff
@@ -166,11 +166,17 @@ def save(
                 channel_colors=channel_colors,
                 dimension_order=dimension_order,
             )
+            xml = self.omeMetadata.to_xml().encode()
+        elif isinstance(ome_xml, str):
+            # if the xml passed in is a string,
+            # then just pass it straight through to the writer.
+            self.omeMetadata = omexml.OMEXML(ome_xml)
+            xml = ome_xml.encode()
         else:
```

Comment:
With the if / elif / else would it be better to have the default action (else) be handle the None case and then have the concrete values (string, whatever valid values for ome_xml are explicitly listed) so that the default is that if it's not a trusted value synthesize the metadata rather than relying on the object having to_xml etc in the else case?

Reply:
yes, I basically agree with you but it's probably not worth the effort as this is going to be totally rewritten very soon for 4.x